### DISCUSSION TITLE
3 New Themes Done First pack! By Momentum and DjRoby

### DISCUSSION THREAD

**Discussion by Onslau** [mathoudebine/turing-smart-screen-python#246] [Themes]
Includes
Cyberdemon Backround DjRob and me
Ultimate Stats - uses all gauges and bars 
Iron Man - yeah simply ironman themed

Have Fun and buy me a Coffe or so XD
[Theme Pack.zip](https://github.com/mathoudebine/turing-smart-screen-python/files/11582926/Theme.Pack.zip)
Instructions included with pictures !

**Comment by Onslau** [comment]
![background](https://github.com/mathoudebine/turing-smart-screen-python/assets/132712795/4a051484-7cd4-4460-81f6-86a2464074e3)

**Reply by Onslau** [reply]
Please use these as backround and put gauges or whatever you want in place or use from DjRobythe yaml file have fun !

**Comment by Onslau** [comment]
![240457010-911139f2-0a92-4746-8cc1-845ca1c1c846](https://github.com/mathoudebine/turing-smart-screen-python/assets/132712795/77e366f3-2e01-42c4-a961-20d87481c324)Ultimate Stat

**Comment by Onslau** [comment]
![237804211-69f9f822-5fb3-4f15-9bd7-25c520775f8c](https://github.com/mathoudebine/turing-smart-screen-python/assets/132712795/e662fba3-2bc5-4779-b189-fad411353a16)
Iron Man

**Comment by Onslau** [comment]
All themes can used and customized as well !

**Comment by Paradoxum** [comment]
Nice thanks :)
